ts/components/dialog/ReactClearAllModal.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ReactElement } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { updateReactClearAllModal } from '../../state/ducks/modalDialog';
+import { StateType } from '../../state/reducer';
+import { getMessageReactsProps } from '../../state/selectors/conversations';
+import { getTheme } from '../../state/selectors/theme';
+import { Flex } from '../basic/Flex';
+import { SessionButton, SessionButtonColor, SessionButtonType } from '../basic/SessionButton';
+import { SessionWrapperModal } from '../SessionWrapperModal';
+
+type Props = {
+  reaction: string;
+  messageId: string;
+};
+
+const StyledReactClearAllContainer = styled(Flex)<{ darkMode: boolean }>`
+  margin: var(--margins-lg);
+
+  p {
+    font-size: 18px;
+    font-weight: bold;
+    padding-bottom: var(--margins-lg);
+    margin: var(--margins-md) auto;
+    border-bottom: 1.5px solid ${props => (props.darkMode ? '#2D2D2D' : '#EEEEEE')};
```

Comment:
I think these colors will be temporary because when we do theming all borders should become 1 color. I just took these colors out of the Figma mockup. Maybe we need to discuss this